include/pros/imu.h

```diff
@@ -465,7 +471,8 @@ int32_t imu_set_pitch(uint8_t port, double target);
 
 /**
  * Sets the current reading of the Inertial Sensor's roll to target value
- *
+ * Will default to +/- 180 if value target +/- 180.
```

Comment:
is this line supposed to be the same as the others?

Reply:
yes, I'm trying to say that if the target value is above or below +180 and -180 respectively, it will default to +180 or -180.

Reply:
right, but the wording here is inexplicably different from the others

Reply:
All three euler values have their bounds at -180 and 180, and each setter function has the same feature.